### PR TITLE
Running programs section on docsite

### PR DESCRIPTION
**Pull request by esthermmoturi**
Documentation site revamp is in progress and should look as highlighted on [this](https://docs.google.com/document/d/16zqhe3OW-zSNoTR1Afm-eUjxYAy73BKlKYB5_XBU3js/edit?usp=sharing) document. Suggestions are welcome.
The goal of this PR is to:

1. Add running program section on the doc-site which highlights requirements of running successful CHT programs
2. Highlighting personas who interact with the CHT
3. Adding the CHT Lifecycle
4. Differentiate between different types of CHT programs

### CODE REVIEW

**Comment by esthermmoturi** [comment]
Hey @antonykhaemba ,kindly share more details on the difference between research initiatives and the TP programs for example difference in duration and deliverables. Also please suggest anything else we need to add.
Hey @garethbowen , kindly share more details on items we champion for.

File: content/en/running-programs/_index.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+8. Closeout - includes budget tracking , discussions on scaling, report writing on impact and challenges experienced.
+
+## Program personas
+Various people interact with CHT programs at various stages.These people include program heads, program administrators,service designers,UX designers,software developers,system administrators and technical support officials.
+
+## Running successful community health programs
+Running a successful digital health program depends on various factors. Here are a couple of things we champion for:
+1. Community Health Workers compensation
+2. Privacy and protection of user data
+
```
Comment (garethbowen) [inline content/en/running-programs/_index.md:50]: There are IMO heaps more to add here, and maybe worth having different sections, eg:

```
### Professional CHWs

<include the list from CHIC>

### Technical

- Privacy and protection of user data
- Monitoring
- Backups
- Redundancy
- Regular CHT upgrades
- Regular device maintenance (Android app and security updates)
- Regular device hardware replacement
- ...
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
This is great, thanks!

Some suggestions inline.

**Comment by n-orlowski** [comment]
CHTLifecycle
 @esthermmoturi LMK if this works for ya

File: content/en/running-programs/_index.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+4. Building - refers to development of forms, tasks, targets, debugging and other aspects of CHT applications.
+5. Hosting - includes different hosting options, data migration, configuration and optimization of CHT applications in various environments.
+6. Implementation - refers to the process of putting CHT applications into action. It includes training plan development, logistics planning, expenditure forecasting and tracking, tech support training, application installation and end user training.
+7. Support - refers to activities done after deployment and implementation. It includes managing updates, building dashboards, monitoring and evaluation, impact measurement and user experience feedback.
+8. Closeout - includes budget tracking , discussions on scaling, report writing on impact and challenges experienced.
+
+## Program personas
+Various people interact with CHT programs at various stages.These people include program heads, program administrators,service designers,UX designers,software developers,system administrators and technical support officials.
```
[the remark this answers is not in the document]
Comment (esthermmoturi) [reply content/en/running-programs/_index.md]: Done!

**Comment by esthermmoturi** [comment]
Currently incorporating feedback provided offline by @antonykhaemba and Simon.

File: content/en/running-programs/_index.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+- Human-centered design
+- Consistent user feedback
+- Material design guidelines
+
+
+
+
+
+
```
Comment (garethbowen) [inline content/en/running-programs/_index.md:93]:
```suggestion

```

File: content/en/running-programs/_index.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+CHT programs are powered by various tools and commodities that are integral.
+
+- Privacy and protection of user data
+- Monitoring and evalution
+- Data backups
+- Redundancy
+- Regular CHT upgrades
+- Regular device maintenance (Android app and security updates)
+- Regular device hardware replacement
+- Open source technologies
```
Comment (garethbowen) [inline content/en/running-programs/_index.md:80]: Not sure this fits...

```suggestion
```

File: content/en/core/_index.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "CHT Core Framework"
 linkTitle: "CHT Core Framework"
-weight: 2
+weight: 3
```
Comment (garethbowen) [inline content/en/core/_index.md:4]: Let's make this 4 so it's clear it comes after "CHT Applications"

```suggestion
weight: 4
```

**Review by garethbowen — CHANGES_REQUESTED** [review]
This is amazing, thank you.

Minor suggestions inline then we're good to go.

File: content/en/running-programs/_index.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+The CHT has a robust community that works together to identify goals that are used to run CHT implementations. Community led implementations emphasize on the importance of collaboration and sustainability.
+As a technical steward of the CHT, Medic has also established a network of local and regional technical organizations that have been onboarded on CHT to enable them to build and support CHT powered applications independently.
+
+**Research initiatives, innovations and studies**
+
+Through human centered design, research institutions and academia  design, test, refine and deploy digital health innovations and carry out research studies. These implementations are aimed at testing concepts or interventions before large scale implementation. They run for a limited time and often have pre-determined objectives to establish facts. 
+
+## Considerations for digital health implementations
+Below are the considerations made when running a digital health implementation:
+- Scope - goals, deadlines, and implementation deliverables
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:25]:
```suggestion
- Scope and project scale, timeliness and project deliverables
```

File: content/en/running-programs/_index.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+- Technical requirements - devices, servers and storage requirements
+- Context - general understanding of the region, policies, legal requirements, mobile connectivity and community health setup
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:27]:
```suggestion
- Context - general understanding of the region, policies, legal requirements, mobile connectivity, community health strategy and existing stakeholders
```

File: content/en/running-programs/_index.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+- Human resource requirements - skills to build and sustain the implementation
+
+## CHT Lifecycle
+CHT implementations follow a process from beginning to end in order to deliver a complete functional solution. This process is referred to as the CHT Lifecycle. Although the process may differ from implementation to implementation, the process below is what has been used for most deployed implementations. It is an agile and iterative process that involves feedback collection and improvement.
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:31]:
```suggestion
CHT implementations follow a process from beginning to end in order to deliver a complete functional digital health solution. This process is referred to as the CHT Lifecycle. Although the process may differ from implementation to implementation, the process below is what has been used for most deployed implementations. It is an agile and iterative process that involves feedback collection and improvement.
```

File: content/en/running-programs/_index.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+{{< figure src="cht_lifecycle.png" link="cht_lifecycle.png" class="center col-8 col-lg-8" >}}
+- Initiation - at this stage, the implementing partner defines the scope of the implementation they would like to run. They also understand the various capabilities and features of the CHT.
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:33]:
```suggestion
- Initiation - at this stage, the project scope, goal and use cases are defined.
```

File: content/en/running-programs/_index.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+- Planning - includes coming up with detailed workplans, schedules, budget estimation, baseline data collection, user estimation and resource planning.
+- Design - design plan sharing and reviewing, design field trips, definition of user personas, preperation of design workflows and implementation hierarchy.
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:35]:
```suggestion
- Design - a design activity is conducted to gather user insights and requirements that will inform the design of the workflows in the CHT.
```

File: content/en/running-programs/_index.md [hunk 11]
```diff
@@ -0,0 +1,82 @@
...
+- Building - refers to the app development process where workflows are configured, tested and improved.
+- Hosting - includes different hosting options, data migration, configuration and optimization of CHT applications in various environments.
+- Implementation - refers to the process of training end users and them using the CHT application in their daily work. It includes training plan development, logistics planning, expenditure estimation and tracking, tech support training and application installation.
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:38]:
```suggestion
- Implementation - refers to the process of training end users and them using the CHT application to support the delivery of health care services. It includes training plan development, logistics planning, expenditure estimation and tracking, tech support training and application installation.
```

File: content/en/running-programs/_index.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+- Support - refers to activities done after deployment and implementation. It includes managing updates, building dashboards, monitoring and evaluation, impact measurement and user experience feedback.
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:39]:
```suggestion
- Support - this is the support provided to end users to address any technical issues and ensure the usability of the tools.
```

File: content/en/running-programs/_index.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+- Evaluation - includes budget tracking, discussions on scaling, report writing and user feedback activities to help improve user experience and impact before the next development cycle begins.
+
+## Implementation personas
+Various people interact with CHT implementations at various stages. The list below may not be exhaustive and different personas can have overlapping roles.
+- Program administrators - they are also referred to as program heads or digital health leads. They are involved in planning, directing and coordinating resources, meetings, payment, key milestones and communicating the workplan and schedule. Although these personas oversee the entire implementation, they interact with the CHT implementation closely at the planning, implementation and evaluation stages. They include implementation managers, finance officers and project managers.
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:44]:
```suggestion
- Project managers - they are also referred to as program heads or program leads. They are involved in planning, directing and coordinating digital health projects. Although these personas oversee the entire implementation, they interact with the CHT implementation closely at the planning, implementation and evaluation stages.
```

File: content/en/running-programs/_index.md [hunk 14]
```diff
@@ -0,0 +1,82 @@
...
+- Designers - they include service designers, UX/UI designers, product designers and graphic designers. Designers interact with CHT implementations at different stages including the design stage where service designers carry out field visits to interact with prospective end users and come up with design specification forms that inform what is required for app development. Product and UI/UX designers are involved in the design phase and evaluation stages where they provide feedback to help optimize user interaction.
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:45]:
```suggestion
- Designers - they include service designers, user experience/user interface designers, product designers and graphic designers. Designers interact with CHT implementations at different stages including the design stage where service designers carry out field visits to interact with prospective end users and come up with design specification forms that inform what is required for app development. Product and UI/UX designers are involved in the design phase and evaluation stages where they provide feedback to help optimize user interaction.
```

File: content/en/running-programs/_index.md [hunk 15]
```diff
@@ -0,0 +1,82 @@
...
+- Developers - they include app developers, core developers and software developers who are invloved in the building stage. App developers build application forms, contact forms, application settings among other configurations. Core devlopers build the [CHT Core framework]({{< ref "core" >}}) that forms the basis of CHT applications.
+- System administrators - they include database administators, security engineers,site reliability engineers, devOps engineers and network managers. They are responsible for the upkeep, uptime and configuration of computers, servers, databases, pipelines and infrastructure of CHT applications. They interact with CHT implementations at the planning, hosting and support stages.
+- Technical support team - they include ICT officers and monitoring & evaluation officers. ICT officers are responsible for installation and technical assistance provision for issues experienced by end users post deployment. Monitoring & evaluation officers are responsible for monitoring project indicators, target evaluation, dashboard monitoring and implementation data collection for analysis and reporting. They interact with CHT implementations at the support stage.
```
Comment (antonykhaemba) [inline content/en/running-programs/_index.md:48]:
```suggestion
- Technical support team - they include information communication and technology (ICT) officers and monitoring & evaluation officers. ICT officers are responsible for installation and technical assistance provision for issues experienced by end users post deployment. Monitoring & evaluation officers are responsible for monitoring project indicators, impact evaluation, data collection for analysis and reporting. They interact with CHT implementations at the design and support stages.
```

**Review by antonykhaemba — CHANGES_REQUESTED** [review]

**Comment by esthermmoturi** [comment]
Hey @garethbowen and @antonykhaemba , I have incorporated all changes suggested including those made offline by Simon.

File: content/en/running-programs/_index.md [hunk 16]
```diff
@@ -0,0 +1,81 @@
...
+
+There are various ways of running Community Health Toolkit implementations. It is important to understand the different models for integrating CHT tools into a community health program as discussed below.
+
+**Accompaniment approach**
+
+In this approach, Medic accompanies partners; including ministries of health, through their digital journey that may include designing, building, deploying and supporting CHT based tools for their unique health system needs. 
+
+**Community led implementations**
+
+This model leverages the robust technical expertise of the CHT community of practice. A network of local and regional technical organizations that have been onboarded on CHT are able to provide design, development, deployment and support services for implementing partners - including ministries of health. This network also contributes their learnings to the community through documentation, reference applications, CHT forum and CHT core improvements. Community led implementations emphasize on the importance of collaboration, local ownership and sustainability.
```
Comment (garethbowen) [inline content/en/running-programs/_index.md:15]: I think we need to be really explicit that community led implementations can happen with no assistance from Medic or the release of the community. Currently this whole paragraph makes it feel like you need a whole lot of technical expertise to pull it off, or you need to work with the network of local technical orgs. How can we reword to make it clear this is a self-led deployment with community support available?

File: content/en/running-programs/_index.md [hunk 17]
```diff
@@ -0,0 +1,81 @@
...
+
+## Considerations for digital health implementations
+To run a successful digital health implementation, the following considerations are made:
+- Scope - Clear definition of the challenge, duration of implementation, implementation deliverables, and the change expected as a result of implementation
+- Technical requirements - infrastructure (servers - physical/ cloud) and hardware (devices such as smartphones/tablets, laptops/computers) requirements
+- Context - general understanding of the geography (country/region) in the following domains; legal and policy landscape, infrastruture penetration e.g. mobile and electricity connectivity, community health setup and maturity e.g cadres of community health workers, coverage and functionality
+- Human resource requirements - skills to build, implement and sustain the implementation
```
Comment (garethbowen) [inline content/en/running-programs/_index.md:26]: Capitalisation is inconsistent here. Also it's easier to scan if you bold the key point, eg:

```suggestion
To run a successful digital health implementation, the following considerations are made:
- **Scope**. Clear definition of the challenge, duration of implementation, implementation deliverables, and the change expected as a result of implementation.
- **Technical requirements**. Infrastructure (servers - physical/ cloud) and hardware (devices such as smartphones/tablets, laptops/computers) requirements.
- **Context**. General understanding of the geography (country/region) in the following domains; legal and policy landscape, infrastructure penetration e.g. mobile and electricity connectivity, community health setup and maturity e.g cadres of community health workers, coverage and functionality.
- **Human resource requirements**. Skills to build, implement and sustain the implementation.
```

File: content/en/running-programs/_index.md [hunk 18]
```diff
@@ -0,0 +1,81 @@
...
+
+**Community health workers**
+
+Community health workers deliver essential health services to the last mile to improve health outcomes. Community health workers require good working conditions and the right tools to perform their responsibilities. To enable this, the items listed below are important. You can learn more on the [eight proCHW best practices](https://joinchic.org/what-we-do/).
+
+- Continuous training for skill growth
+- Supportive supervision for optimal performance management
+- Compensation for economic growth and sustainability
+- Institutionalization into the formal health workforce
+- Equipping with required supplies and kits
```
Comment (garethbowen) [inline content/en/running-programs/_index.md:62]: Let's enforce the CHIC 4S model here..

```suggestion
- **Skilled**. Continuous training for skill growth
- **Supervised**. Supportive supervision for optimal performance management
- **Salaried**. Compensation for economic growth and sustainability
- **Supplied**. Equipping with required supplies and kits
```

File: content/en/running-programs/_index.md [hunk 19]
```diff
@@ -0,0 +1,81 @@
...
+
+**Technical performance optimization**
+
+For CHT powered tools to perform optimally, the following requirements are integral.
```
Comment (garethbowen) [inline content/en/running-programs/_index.md:66]: Some of these are less about "tools" and more about the programs themselves.

```suggestion
For CHT implementations to be successful and sustainable, the following requirements are integral.
```

**Review by garethbowen — CHANGES_REQUESTED** [review]

File: content/en/running-programs/_index.md [hunk 20]
```diff
@@ -0,0 +1,80 @@
...
+
+There are various ways of running Community Health Toolkit implementations. It is important to understand the different models for integrating CHT tools into a community health program as discussed below.
+
+**Accompaniment approach**
+
+In this approach, Medic accompanies partners; including ministries of health, through their digital journey that may include designing, building, deploying and supporting CHT based tools for their unique health system needs. 
+
+**Community led implementations**
+
+This model leverages the robust technical expertise of the CHT community of practice. A network of local and regional technical organizations that have been onboarded on CHT are able to provide design, development, deployment and support services for implementing partners - including ministries of health. This network also contributes their learnings to the community through documentation, reference applications, CHT forum and CHT core improvements. Community led implementations emphasize on the importance of collaboration, local ownership and sustainability.
```
Comment (esthermmoturi) [inline content/en/running-programs/_index.md:15]:
```suggestion
Community led implementations emphasize on the importance of collaboration, local ownership and sustainability. This is a self-led model where a community identifies a digital health problem and solves it using CHT tools. The CHT also has a community of practice that is made up of a network of local and regional technical organizations that have been onboarded on CHT and are able to provide design, development, deployment and support services for implementing partners - including ministries of health. This network also contributes their learnings to the community through documentation, reference applications, CHT forum and CHT core improvements. 
```

Comment (esthermmoturi) [reply content/en/running-programs/_index.md:15]: @garethbowen ,is this okay or should I not mention the community of practice and technical organisations completely?

Comment (garethbowen) [reply content/en/running-programs/_index.md:15]: I think it's perfect, because you start with self-led and go into the options for support. Thanks!

**Review by garethbowen — APPROVED** [review]
Awesome!

**Review by antonykhaemba — APPROVED** [review]

**Pull request merged**